ssl-monitor/api/alert_service.py

```
@@ -0,0 +1,304 @@
+    """
+    if not webhook_url:
+        return False
+    
+    try:
+        response = requests.post(
+            webhook_url,
+            json=alert_data,
+            headers={'Content-Type': 'application/json'},
+            timeout=10
+        )
+        return response.status_code in [200, 201, 202, 204]
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)